docs/api.md

```diff
@@ -81,7 +94,73 @@ $ curl --user john.doe@example.com:PASSWORD http://localhost:8100/whoami
 < {"username":"john.doe@example.com","scopes":["authenticated","example_scope"]}
 ```
 
-### Forwarding statements
+### OpenID Connect authentication
+
+Ralph LRS API server supports OpenID Connect (OIDC) on top of OAuth 2.0 for authentication and authorization.
+
+To enable OIDC auth, you should set the `RALPH_RUNSERVER_AUTH_BACKEND` environment variable to `oidc`, and you should define the `RALPH_RUNSERVER_AUTH_OIDC_ISSUER_URI` environment variable with your identity provider's Issuer Identifier URI as such:
```

Comment:
I suggest to have a more visual documentation for setting environment variables. User just have to copy/paste the proposed configuration (and adapt it if necessary)

```suggestion
To enable OIDC auth, you should set the `RALPH_RUNSERVER_AUTH_BACKEND` environment variable as follows:
`
RALPH_RUNSERVER_AUTH_BACKEND=oidc

and you should define the `RALPH_RUNSERVER_AUTH_OIDC_ISSUER_URI` environment variable with your identity provider's Issuer Identifier URI as follows:
`
RALPH_RUNSERVER_AUTH_OIDC_ISSUER_URI=http://{provider_host}:{provider_port}/auth/realms/{realm_name}
`
```